spec/support/stub.rb

```diff
@@ -0,0 +1,9 @@
+RSpec.configure do |config|
+  config.before :suite do
+    ApplicationHelper.module_eval do
+      def annict_image_url(record, field, options = {})
```

Comment:
Unused method argument - `record`. If it's necessary, use `_` or `_record` as an argument name to indicate that it won't be used. You can also write as `annict_image_url(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `field`. If it's necessary, use `_` or `_field` as an argument name to indicate that it won't be used. You can also write as `annict_image_url(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `annict_image_url(*)` if you want the method to accept any arguments but don't care about them.